### PR TITLE
add DSL's for initial review

### PR DESCRIPTION
**Pull request by BRMcLaren**
Signed-off-by: brmclaren <brmclare@microsoft.com>

### CODE REVIEW

File: config/jenkins/configuration/jobs/oeedger8r-cpp/Ubuntu1604BuildClang-7.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            parameters {
+                stringParam {
+                    name("PULL_NUMBER")
+                    defaultValue("master")
+                    description("")
+                    trim(true)
+                }
+                stringParam {
+                    name("LINUX_VERSION")
+                    defaultValue("1804")
```
Comment (CyanDevs) [inline config/jenkins/configuration/jobs/oeedger8r-cpp/Ubuntu1604BuildClang-7.yml:19]: Is this meant for 1804 or 1604?
Your file is 1604, but job name and param is 1804.

Comment (CyanDevs) [reply config/jenkins/configuration/jobs/oeedger8r-cpp/Ubuntu1604BuildClang-7.yml:19]: Same with the other file

File: config/jenkins/configuration/jobs/test-infra/oeedger8r-cpp/Ubuntu1604BuildClang-7.yml [hunk 2]
```diff
@@ -0,0 +1,80 @@
+---
+jobs:
+  - script: |
+        folder("test-infra") {
+            displayName("test-infra")
+        }
+        folder("test-infra/oeedger8r-cpp") {
+            displayName("test-infra/oeedger8r-cpp")
+        }
+        pipelineJob("test-infra/oeedger8r-cpp/Ubuntu1804BuildClang-7) {
```
Comment (CyanDevs) [inline config/jenkins/configuration/jobs/test-infra/oeedger8r-cpp/Ubuntu1604BuildClang-7.yml:10]: Missing end double quotes

File: config/jenkins/configuration/jobs/oeedger8r-cpp/Ubuntu1604BuildClang-7.yml (hunk 1 above)
Comment (BRMcLaren) [reply config/jenkins/configuration/jobs/oeedger8r-cpp/Ubuntu1604BuildClang-7.yml]: Oh the joys of refactoring...

Thanks, changed!

**Pull request merged**